### ISSUE TITLE
I do not found a cli (command line) script to test, there any avaliable?

### ISSUE THREAD

**Issue by fernandobhz** [wuhy/inline-resource#1]
I do not found a cli (command line) script, there any avaliable?

That will be helpfull to use on non node projects, so .net, php and java can use your script.

If you don't know see: https://developer.atlassian.com/blog/2015/11/scripting-with-node/

**Comment by wuhy** [comment]
Yes, CLI support is not provided in this tool currently. I'll consider to provide this, thanks for your suggestion.
